starboard/decode_target.h

```diff
@@ -252,7 +252,7 @@ typedef struct SbDecodeTargetInfo {
   int height;
 
   // The image planes (e.g. kSbDecodeTargetPlaneRGBA, or {kSbDecodeTargetPlaneY,
-  //  kSbDecodeTargetPlaneU, kSbDecodeTargetPlaneV} associated with this
+  // kSbDecodeTargetPlaneU, kSbDecodeTargetPlaneV}) associated with this
```

Comment:
This parenthesis look incorrect. Why is it added?

Reply:
This closes the parenthesis started at `e.g.`.